x-pack/plugins/security_solution/public/management/pages/policy/types.ts

```diff
@@ -23,6 +25,17 @@ import {
 } from '../../../../../fleet/common';
 import { AsyncResourceState } from '../../state';
 import { TrustedAppsListData } from '../trusted_apps/state';
+import { ImmutableMiddlewareAPI } from '../../../common/store';
+import { AppAction } from '../../../common/store/actions';
+
+/**
+ * Function that runs Policy Details middleware
+ */
+export type MiddlewareRunner = (
```

Comment:
Nitpick: This should be named something like `PolicyMiddlewareRunner` as it looks like it's not a generic "MiddlewareRunner"